lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -300,9 +300,10 @@ def _on_leader_elected(self, event: LeaderElectedEvent):
 
         self.status.clear(AdminUserInitProgress)
 
-    def _on_start(self, event: StartEvent):
+    def _on_start(self, event: StartEvent):  # noqa C901
         """Triggered when on start. Set the right node role."""
-        if self.opensearch.is_node_up():
+
+        def __on_start_post_cleanup():
```

Comment:
nit - given its within a method, we can get away with a simple name, e.g: `cleanup`